mmeds/tests/unit/test.py

```diff
@@ -102,6 +102,15 @@ def setup_tests(tests):
                         fig.TEST_OTU,
                         fig.TEST_CODE_OTU)
             assert 0 == upload_otu(test_otu)
+            #Upload Lefse data if running test_tools.py
```

Comment:
Spaces before comments
```suggestion
            # Upload Lefse data if running test_tools.py
```